src/main/steps/respondent/task-list/tasklist.ts

```diff
@@ -50,8 +51,8 @@ export const generateRespondentTaskList = (sectionTitles, taskListItems, userCas
         {
           id: 'check_details_of_your_court_hearings',
           text: taskListItems.check_details_of_your_court_hearings,
-          status: getInternationalFactorsStatus(userCase),
-          href: URL.INTERNATIONAL_FACTORS_START,
+          status: getViewAllHearingsFromTheCourt(userCase),
+          href: URL.RESPONDENT_YOURHEARINGS_HEARINGS,
```

Comment:
??

Reply:
This is required for respondent hearings